javaassignment.js

```diff
@@ -0,0 +1,381 @@
+/*question # 1*/
+function ph(i){
+    if(i>7){
+        return 'acidic';
+    }else if (i<7){
+        return 'basic';
+    }else if (i=7) {
+        return 'neutral';
+    } else {
+        return 'undefined';
+    }
+ }
+ /*
+ console.log('the out put for 7 is  ' + ph(7))
+ console.log('the out put for 6 is  ' + ph(6))
+ console.log('the out put for 3 is  ' + ph(3));
+PS C:\Users\bisrat\Desktop\source> node test.js
+the out put for 7 is  neutral
+the out put for 6 is  basic
+the out put for 3 is  basic 
+
+*****************************************************************
+
+ question #2 */
+
+function dis(price){
+ if(price >=300){
+    price=price-(price*(50/100));
+    console.log(price);
+}else if (200<price<300){
+    price=price-(price*(20/100));
+    console.log(price);
+}else if (0<price<=200){
+    price=price-(price*(5/100));
+    console.log(price); 
+}else
+{
+    console.log('error');
+}
+ }
+ 
+ console.log(dis(300));
+/*
+PS C:\Users\bisrat\Desktop\source> node assignement.js
+150
+   */
+/***************************************************************************************
+
+/*question #3 */
+function shi(price, location){
+    if(location == 'EU')
+    {
+        let totalcost ;
+        totalcost=price+5;
+        return 'you have to pay totalcost of ' +totalcost+ ", "+ price +' '+'for the product and 5 for the shipping';
+    }
+    else if (location == "US" || location =="Canada"){
+        let totalcost ;
+        totalcost=price+15;
+        return 'you have to pay totalcost of ' +totalcost+ ", "+ price +' '+'for the product and 5 for the shipping';
+    }else{
+        let totalcost ;
+        totalcost=price+20;
+        return 'you have to pay totalcost of ' +totalcost+ ", "+ price +' '+'for the product and 5 for the shipping';
+    }
+}
+
+console.log(shi(25,'US') );
+
+/*PS C:\Users\bisrat\Desktop\source> node asspra
+you have to pay totalcost of 30 ,  25for the product and 5 for the shipping
+PS C:\Users\bisrat\Desktop\source> node asspra
+you have to pay totalcost of 40, 25 for the product and 5 for the shipping */
+
+/*************************************************************************************
+
+/*question #4 */
+function litsong(){
+    let i=10
+    for(i=10;i>0;i--){
+        let y =i-1;
+        console.log(i+" "+"in the bed, the little one said roll over " + y +" leff off");
+    }
+    console.log ('The little one said, Everybody back on the bed!');
+}
+
+litsong();
+/*
+PS C:\Users\bisrat\Desktop\source> node asspra
+10 in the bed, the little one said roll over 9 leff off

```

Comment:
9 fell off? should be only 1 fell off 😄  if 10 in the bed and 9 fell off, next loop would show 1 in the bed..